feeds/pypi/main.go

```diff
@@ -17,16 +27,25 @@ type Response struct {
 }
 
 type Package struct {
+	Title        string      `xml:"title"`
 	ModifiedDate rfc1123Time `xml:"pubDate"`
 	Link         string      `xml:"link"`
-	Name         string      `xml:"-"`
-	Version      string      `xml:"-"`
 }
 
 type rfc1123Time struct {
 	time.Time
 }
 
+func (p *Package) Name() string {
```

Comment:
I'm good with this! One of the things that took me a bit was that I originally tried to be clever and do a custom unmarshaler to split the link, but that turned into a self-referential infinite loop, which would have required a type alias, etc. It was all very gross so I figured "eh, let's just parse it then split it" and this follows that same idea. So 👍 from me!

Reply:
Ah interesting. Yeah I was getting empty values and this was the fastest way I could think of. The alias/infinite recursion problem makes sense now though. Thanks for saving me the time!